tests/apitest/src/AppKit/NSWorkspace.cs

```diff
@@ -22,5 +22,201 @@ public void NSWorkspaceConstantTests ()
+			Assert.That(FourCC((int)HfsTypeCode.ToolbarPicturesFolderIcon), Is.EqualTo("tPic"), "tPic");
+			Assert.That(FourCC((int)HfsTypeCode.ToolbarPublicFolderIcon), Is.EqualTo("tPub"), "tPub");
+			Assert.That(FourCC((int)HfsTypeCode.ToolbarDesktopFolderIcon), Is.EqualTo("tDsk"), "tDsk");
+			Assert.That(FourCC((int)HfsTypeCode.ToolbarDownloadsFolderIcon), Is.EqualTo("tDwn"), "tDwn");
+			Assert.That(FourCC((int)HfsTypeCode.ToolbarLibraryFolderIcon), Is.EqualTo("tLib"), "tLib");
+			Assert.That(FourCC((int)HfsTypeCode.ToolbarUtilitiesFolderIcon), Is.EqualTo("tUtl"), "tUtl");
+			Assert.That(FourCC((int)HfsTypeCode.ToolbarSitesFolderIcon), Is.EqualTo("tSts"), "tSts");
+
+			/* Other icons */
+			Assert.That(FourCC((int)HfsTypeCode.AppleLogoIcon), Is.EqualTo("capl"), "capl");
+			Assert.That(FourCC((int)HfsTypeCode.AppleMenuIcon), Is.EqualTo("sapl"), "sapl");
+			Assert.That(FourCC((int)HfsTypeCode.BackwardArrowIcon), Is.EqualTo("baro"), "baro");
+			Assert.That(FourCC((int)HfsTypeCode.FavoriteItemsIcon), Is.EqualTo("favr"), "favr");
+			Assert.That(FourCC((int)HfsTypeCode.ForwardArrowIcon), Is.EqualTo("faro"), "faro");
+			Assert.That(FourCC((int)HfsTypeCode.GridIcon), Is.EqualTo("grid"), "grid");
+			Assert.That(FourCC((int)HfsTypeCode.HelpIcon), Is.EqualTo("help"), "help");
+			Assert.That(FourCC((int)HfsTypeCode.KeepArrangedIcon), Is.EqualTo("arng"), "arng");
+			Assert.That(FourCC((int)HfsTypeCode.LockedIcon), Is.EqualTo("lock"), "lock");
+			Assert.That(FourCC((int)HfsTypeCode.NoFilesIcon), Is.EqualTo("nfil"), "nfil");
+			Assert.That(FourCC((int)HfsTypeCode.NoFolderIcon), Is.EqualTo("nfld"), "nfld");
+			Assert.That(FourCC((int)HfsTypeCode.NoWriteIcon), Is.EqualTo("nwrt"), "nwrt");
+			Assert.That(FourCC((int)HfsTypeCode.ProtectedApplicationFolderIcon), Is.EqualTo("papp"), "papp");
+			Assert.That(FourCC((int)HfsTypeCode.ProtectedSystemFolderIcon), Is.EqualTo("psys"), "psys");
+			Assert.That(FourCC((int)HfsTypeCode.RecentItemsIcon), Is.EqualTo("rcnt"), "rcnt");
+			Assert.That(FourCC((int)HfsTypeCode.ShortcutIcon), Is.EqualTo("shrt"), "shrt");
+			Assert.That(FourCC((int)HfsTypeCode.SortAscendingIcon), Is.EqualTo("asnd"), "asnd");
+			Assert.That(FourCC((int)HfsTypeCode.SortDescendingIcon), Is.EqualTo("dsnd"), "dsnd");
+			Assert.That(FourCC((int)HfsTypeCode.UnlockedIcon), Is.EqualTo("ulck"), "ulck");
+			Assert.That(FourCC((int)HfsTypeCode.ConnectToIcon), Is.EqualTo("cnct"), "cnct");
+			Assert.That(FourCC((int)HfsTypeCode.GenericWindowIcon), Is.EqualTo("gwin"), "gwin");
+			Assert.That(FourCC((int)HfsTypeCode.QuestionMarkIcon), Is.EqualTo("ques"), "ques");
+			Assert.That(FourCC((int)HfsTypeCode.DeleteAliasIcon), Is.EqualTo("dali"), "dali");
+			Assert.That(FourCC((int)HfsTypeCode.EjectMediaIcon), Is.EqualTo("ejec"), "ejec");
+			Assert.That(FourCC((int)HfsTypeCode.BurningIcon), Is.EqualTo("burn"), "burn");
+			Assert.That(FourCC((int)HfsTypeCode.RightContainerArrowIcon), Is.EqualTo("rcar"), "rcar");
```

Comment:
All lines in this method:

```suggestion
			Assert.That (FourCC ((int) HfsTypeCode.RightContainerArrowIcon), Is.EqualTo ("rcar"), "rcar");
```